java/src/main/java/com/xxAMIDOxx/xxSTACKSxx/menu/handlers/CreateItemHandler.java

```diff
@@ -58,9 +57,7 @@ Category getCategory(Menu menu, CreateItemCommand command) {
         () -> new CategoryDoesNotExistException(command, command.getCategoryId()));
   }
 
-  Category addItem(Category category, CreateItemCommand command) {
-
-    itemId = UUID.randomUUID();
+  Category addItem(Category category, CreateItemCommand command, UUID itemId) {
```

Comment:
again we should be able to simply pass the category and command here and can remove the itemId from the variables